conda_smithy/templates/README.md.tmpl

```diff
@@ -60,7 +60,13 @@ Installing `{{ package.name() }}` from the `{{ channel_name }}` channel can be a
 conda config --add channels {{ channel_name }}
 ```
 
-Once the `{{ channel_name }}` channel has been enabled, `{{ package.name() }}` can be installed with:
+Once the `{{ channel_name }}` channel has been enabled, update `conda` with
+
+```
+conda update conda
```

Comment:
@jakirkham, should this have `-n root` here?

Reply:
If you want to be explicit, sure. Though `conda` can't be installed in a non-`root` environment (or at least not easily). So I think it follows from what we are asking.

Reply:
Though generally asking people to update `conda` could bite us when `conda` 4.4.0 lands.

Reply:
> Though conda can't be installed in a non-root environment 

Yes, but if you are on a env, it will give an error.

> Though generally asking people to update conda could bite us when conda 4.4.0 lands.

Not if we don't add `conda=4.4.0` to `conda-forge`

Reply:
For the release 2.4.1, should we add this or tag the commit before this PR was merged?

Reply:
@jakirkham, what do you think?

Reply:
Sorry, only coming to this now. Fine by me either way. I trust your judgement.